.github/workflows/main.yml

```diff
@@ -0,0 +1,324 @@
+# --------------------------------------------
```

Comment:
Why is that file shown as new here? Didn't we merge the last releas also onto master?

Reply:
It got merged to the master/LTS for 4.12.x and not the master.